app/models/bot/smooch.rb

```diff
@@ -600,6 +600,7 @@ def self.save_message(message_json, app_id)
          when 'video'
            self.save_media_message(message, 'video')
          when 'file'
+           self.detect_media_type(message)
            m = message['mediaType'].to_s.match(/^(image|video)\//)
```

Comment:
question to @melsawy and @caiosba : for security reasons, should we reject, or at least log, files whose MIME type differs between our detection and Smooch? Or are we not worried about their contents?

Reply:
I think we should log (to Errbit) but still save it

Reply:
I agree to log, but I'd rather use the regular log and use a WARNING level. Maybe we need to agree on log vs. Errbit.

Reply:
For now I'll use a regular log with WARNING is this OK?

Reply:
sounds good